tests/suites/0_stateless/13_tpch/13_0009_q9.sql

```diff
@@ -2,7 +2,7 @@ set enable_planner_v2 = 1;
 select
     nation,
     o_year,
-    sum(amount) as sum_profit
+    to_int64(sum(amount)) as sum_profit
```

Comment:
I just notice that we have `truncate` function, using it can have a minor difference with PG.

```
databend > select truncate(3.12345, 3);
+----------------------+
| truncate(3.12345, 3) |
+----------------------+
|                3.123 |
+----------------------+
1 row in set (0.003 sec)
```

Reply:
ok,is it appropriate to keep three digits? I see that the result of pg has more digits after the decimal point.

Reply:
You can try to set more digits.

Reply:
ok,thanks!